modules/tinymce/src/core/main/ts/fmt/MatchFormat.ts

```diff
@@ -8,13 +8,14 @@
 import { Arr } from '@ephox/katamari';
 import DOMUtils from '../api/dom/DOMUtils';
 import Editor from '../api/Editor';
-import { Format, FormatVars, SelectorFormat } from '../api/fmt/Format';
+import { Format, FormatVars, SelectorFormat } from './FormatTypes';
 import * as FormatUtils from './FormatUtils';
 
 const isEq = FormatUtils.isEq;
 
 const matchesUnInheritedFormatSelector = function (ed: Editor, node: Node, name: string) {
-  const formatList = ed.formatter.get(name);
+  // TODO: Is this safe? it doesn't look like it is this could be a block or inline format
```

Comment:
Can we Jira this up as an investigation so it doesn't get forgotten?

Reply:
Formatter has so many issues and it's known it doesn't type check. I don't want a JIRA for this single specific case.

Reply:
Worth noting @ltrouton and I discussed this more and we verified it is an actual bug, though requires invalid configuration to achieve it: http://fiddle.tinymce.com/Auhaab. I'll log a JIRA for that since we now know it is a bug.